DQM/SiTrackerPhase2/python/Phase2TrackerDQMFirstStep_cff.py

```diff
@@ -3,10 +3,23 @@
 from DQM.SiTrackerPhase2.Phase2ITMonitorRecHit_cff import *
 from DQM.SiTrackerPhase2.Phase2ITMonitorCluster_cff import *
 from DQM.SiTrackerPhase2.Phase2OTMonitorCluster_cff import *
+from DQM.SiTrackerPhase2.Phase2OTMonitorVectorHits_cff import *
 
 trackerphase2DQMSource = cms.Sequence( pixDigiMon 
                                        + otDigiMon
                                        +rechitMonitorIT
                                        + clusterMonitorIT
                                        + clusterMonitorOT
 )
+
+from Configuration.ProcessModifiers.vectorHits_cff import vectorHits
+trackerphase2DQMSource_wVhits = cms.Sequence( pixDigiMon
+                                       + otDigiMon
+                                       +rechitMonitorIT
+                                       + clusterMonitorIT
+                                       + clusterMonitorOT
+                                       + acceptedVecHitsmon
+                                       + rejectedVecHitsmon
+)
```

Comment:
```suggestion
trackerphase2DQMSource_wVhits = trackerphase2DQMSource.copy()
trackerphase2DQMSource_wVhits += cms.Sequence(acceptedVecHitsmon +rejectedVecHitsmon)
```

is more compact and should be equivalent.